src/game.cpp

```diff
@@ -2208,6 +2216,9 @@ int game::inventory_item_menu( int pos, int iStartX, int iWidth,
                 case 'D':
                     u.disassemble( pos );
                     break;
+                case 'f':
+                    u.favorite( pos );
```

Comment:
As oThisItem is still in scope, can't we just toggle is_favorite field directly instead of going through the player?